test/week3Spec.hs

```diff
@@ -0,0 +1,28 @@
+module week3Spec (spec) where
```

Comment:
Haskell module names have to start with uppercase letter.
The name of the haskell file should match the module definition.
```suggestion
module Week3Spec (spec) where
```